sdk/core/Azure.Core/src/Serialization/ModelSerializerFormat.cs

```diff
@@ -0,0 +1,75 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.ComponentModel;
+
+namespace Azure.Core.Serialization
+{
+    /// <summary>
+    /// A format used to specify how a model should be serialized and deserialized.
+    /// </summary>
+    public readonly partial struct ModelSerializerFormat : IEquatable<ModelSerializerFormat>
+    {
+        internal const string JsonValue = "J";
```

Comment:
What happens to all the models that implement the serializable interfaces when we add a format to this?  Do they throw if it's not supported?  For third-party implementors, how should they be thinking about this?  Should they be careful not to introduce custom strings for their own use that we might end up supporting in Azure.Core later?

Reply:
Yes a model should throw if the format is not supported.  Customers can add their own custom strings that their models understand but ours will throw if they receive that custom format.

Reply:
we added [this helper method](https://github.com/Azure/azure-sdk-for-net/pull/35946/files#diff-0a9e09d662352adeada1d1ba9e8f1622695ba0de1419f6957484e0c2bfc47904R13) which our models will use to validate format